uwh-refbox/src/main.rs

```diff
@@ -140,53 +140,259 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error>> {
+                    ),
+                    GamePeriod::FirstHalf => display.draw(
+                        Font8x15::render_str("1ST HALF")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(100, 2)),
+                    ),
+                    GamePeriod::HalfTime => display.draw(
+                        Font8x15::render_str("HALF-TIME")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(92, 2)),
+                    ),
+                    GamePeriod::SecondHalf => display.draw(
+                        Font8x15::render_str("2ND HALF")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(100, 2)),
+                    ),
+                    GamePeriod::PreOvertime => display.draw(
+                        Font8x15::render_str("PRE-OVERTIME")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(80, 2)),
+                    ),
+                    GamePeriod::OvertimeFirstHalf => display.draw(
+                        Font8x15::render_str("O/T 1ST HALF")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(80, 2)),
+                    ),
+                    GamePeriod::OvertimeHalfTime => display.draw(
+                        Font8x15::render_str("O/T HALF TIME")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(76, 2)),
+                    ),
+                    GamePeriod::OvertimeSecondHalf => display.draw(
+                        Font8x15::render_str("O/T 2ND HALF")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(80, 2)),
+                    ),
+                    GamePeriod::PreSuddenDeath => display.draw(
+                        Font8x15::render_str("PRE-SUDDEN DEATH")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(64, 2)),
+                    ),
+                    GamePeriod::SuddenDeath => display.draw(
+                        Font8x15::render_str("SUDDEN DEATH")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(80, 2)),
+                    ),
+                    _ => display.draw(
+                        Font8x15::render_str("ERROR_1")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(64, 2)),
+                    ),
+                }
+            }
+            _ => {
+                display.draw(
+                    Font16x31::render_str(&secs_to_time_string(state.secs_in_period))
+                        .stroke(Some(game_color))
+                        .translate(Point::new(108, 33)),
+                );
+                match state.current_period {
```

Comment:
This can be shortened similarly to the above. Esp necessary with the next comment

Reply:
ok